compiler/src/dotty/tools/dotc/core/tasty/CommentUnpickler.scala

```diff
@@ -20,12 +20,9 @@ class CommentUnpickler(reader: TastyReader) {
     while (!isAtEnd) {
       val addr = readAddr()
       val length = readNat()
-      if (length > 0) {
-        val bytes = readBytes(length)
-        val position = new Span(readLongInt())
```

Comment:
This seems like a bug. If the comment is empty we should read this long int, otherwise the next comment will start by reading this value instead of the the length of the comment. 

I assume it was fine because we never pickled empty documentation. I wonder what should be the behaviour of `/***/`. In that case we still have some coordinates we should pickle.